### PR TITLE
Add translations for buttons and error string on the auth page

### PR DESCRIPTION
**Pull request by bieniu**
## Breaking change

None

## Proposed change

This PR adds translations for buttons `Next` and `Start over` and for error string on the onboarding page.

## Type of change

- [ ] Dependency upgrade
- [X] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue:
- Link to documentation pull request:

## Checklist

- [X] The code change is tested and works locally.
- [X] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/translations/en.json [hunk 1]
```diff
@@ -2037,6 +2037,9 @@
         "form": {
           "working": "Please wait",
           "unknown_error": "Something went wrong",
+          "next": "Next",
+          "start_over": "Start over",
+          "error": "Error:",
```
Comment (bramkragten) [inline src/translations/en.json:2042]: Keep the `:` out of the translation.

Comment (bieniu) [reply src/translations/en.json]: I think I did it wrong. It should be so `"error": "Error: {ErrorMessage}",`. What do you think?

File: src/auth/ha-auth-flow.ts [hunk 2]
```diff
@@ -94,8 +94,11 @@ class HaAuthFlow extends litLocalizeLiteMixin(LitElement) {
       case "error":
         return html`
           <div class="error">
-            ${this.localize("ui.panel.page-authorize.form.error")}
-            ${this._errorMessage}
+            ${this.localize(
+              "ui.panel.page-authorize.form.error",
+              "errorMessage",
```
Comment (bramkragten) [inline src/auth/ha-auth-flow.ts:99]:
```suggestion
              "error",
```

File: src/translations/en.json [hunk 3]
```diff
@@ -2039,7 +2039,7 @@
           "unknown_error": "Something went wrong",
           "next": "Next",
           "start_over": "Start over",
-          "error": "Error:",
+          "error": "Error: {ErrorMessage}",
```
Comment (bramkragten) [inline src/translations/en.json:2042]:
```suggestion
          "error": "Error: {error}",
```

File: src/translations/en.json (hunk 1 above)
Comment (bramkragten) [reply src/translations/en.json]: That's also possible although not really needed here as I think the order will stay the same in all languages.

Comment (bieniu) [reply src/translations/en.json]: Do you want me to change this? The PR can be merged?

**Review by bramkragten — APPROVED** [review]

**Pull request merged**